src/assisted_installer_controller/assisted_installer_controller.go

```diff
@@ -473,6 +496,39 @@ func (c controller) postInstallConfigs(ctx context.Context) error {
 	return nil
 }
 
+func (c controller) getReadyOperators() ([]string, []models.MonitoredOperator, error) {
+	var readyOperators []string
+	operators, err := c.ic.GetClusterMonitoredOLMOperators(context.TODO(), c.ClusterID)
```

Comment:
list of operators can't be changed, why not to provide it once?

Reply:
Right, we can save some assisted-service API calls.